src/data-models/mdm-data-class.model.ts

```diff
@@ -59,7 +59,7 @@ export interface DataClassNode {
   /**
    * The child data classes under this Data Class.
    */
-  dataClasses?: DataClassNode[];
+  dataClasses?: DataClassFull[];
```

Comment:
Could you explain why this change was made? It's not clear to me why this was done.

Reply:
The original caused a compilation error downstream when used. I'm not sure how this passed because the type would have been incorrect self-referencing itself. This fix resolves that.

Reply:
Happy with explanation.